homeassistant/components/wilight/config_flow.py

```diff
@@ -16,7 +16,7 @@
 WILIGHT_MANUFACTURER = "All Automacao Ltda"
 
 # List the components supported by this integration.
-ALLOWED_WILIGHT_COMPONENTS = ["light"]
+ALLOWED_WILIGHT_COMPONENTS = ["light", "fan"]
```

Comment:
This list is used in __init__.py as well. I think it is a good practice to put this in const.py.